### PR TITLE
Added info for public workspaces 

### PR DESCRIPTION
**Pull request by arlemi**
I've added a couple of line in the collaboration section for managing workspaces, will add more as we add new features.

### CODE REVIEW

**Comment by SueSmith** [comment]
@arlemi I'm confused, is the stuff in this PR publicly available to users right now? Or is this for a future release?

**Comment by arlemi** [comment]
@SueSmith My understanding was that it was going to be available today but looks like I was wrong...

**Comment by SueSmith** [comment]
😂 I am so confused rn

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md [hunk 1]
```diff
@@ -43,15 +43,15 @@ You can [create workspaces](/docs/collaborating-in-postman/using-workspaces/crea
 
 ## Accessing workspaces
 
-To see your workspace details, from the Postman app workspace selector, hover over a workspace, click **...**, and select **View Details**.
+You can access Personal, Team, and Private workspaces using the Workspace selector at the top of the Postman app.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:46]: I might not capitalize "Workspace selector"

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md [hunk 2]
```diff
@@ -43,15 +43,15 @@ You can [create workspaces](/docs/collaborating-in-postman/using-workspaces/crea
...
 
-[![view details app](https://assets.postman.com/postman-docs/docs6.1update/Screen+Shot+2018-05-09+at+2.34.21+PM.png)](https://assets.postman.com/postman-docs/docs6.1update/Screen+Shot+2018-05-09+at+2.34.21+PM.png)
+To see and edit your workspace details (e.g. name, description...), from the Postman app workspace selector, hover over a workspace, click **...**, and select **View Details**. You can also view the details and manage your workspaces from the [Workspaces dashboard](https://app.getpostman.com/dashboard).
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:48]: I might adjust the first sentence to something like 

`To view and edit your workspace details (e.g. name, description), navigate to the workspace selector and hover over a workspace. Then, click **...** and select **View Details**.`

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md [hunk 3]
```diff
@@ -43,15 +43,15 @@ You can [create workspaces](/docs/collaborating-in-postman/using-workspaces/crea
...
 
-You can also view the details and manage your workspaces from the [Workspaces dashboard](https://app.getpostman.com/dashboard).
+[![view details app](https://assets.postman.com/postman-docs/docs6.1update/Screen+Shot+2018-05-09+at+2.34.21+PM.png)](https://assets.postman.com/postman-docs/docs6.1update/Screen+Shot+2018-05-09+at+2.34.21+PM.png)
 
-[![Delete workspace menu](https://assets.postman.com/postman-docs/delete-workspace-menu.jpg)](https://assets.postman.com/postman-docs/delete-workspace-menu.jpg)
+You can access Public workspaces from the [**Workspace** category in the API Network](https://explore.postman.com/workspaces).
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:52]: are we branding "Public workspaces"/have we decided on a capitalization? May want to ask Brandon if we should add to https://postmanlabs.atlassian.net/l/c/9eiXG281 and we can update mentions in this doc/others for consistency

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:52]: `**Workspace** category` -> `**Workspaces** category`

File: src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md (hunk 1 above)
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/using-workspaces/managing-workspaces.md:46]: I'd probably also remove capitals from "Personal, Team, and Private" as they aren't capitalized earlier in the page

**Review by claireriley — CHANGES_REQUESTED** [review]
Few small items to look into before publishing (which I assume will be a bit later due to the segmented release but 🤷‍♀️ )

**Pull request closed without merging**